tests/src/mods/auction.rs

```diff
@@ -171,22 +231,67 @@ impl AuctionContract {
         let contract_hash: ContractHash = query(
             builder,
             Key::Account(*admin),
-            &["NFT_contract_hash_wrapped".to_string()],
+            &[[KEY_CEP_47_CONTRACT_HASH, WRAPPED].join("_")],
         );
         let contract_package: ContractPackageHash = query(
             builder,
             Key::Account(*admin),
-            &["NFT_package_hash_wrapped".to_string()],
+            &[[KEY_CEP_47_PACKAGE_HASH, WRAPPED].join("_")],
         );
         (contract_hash, contract_package)
     }
 
+    fn deploy_enhanced_nft(
+        builder: &mut InMemoryWasmTestBuilder,
+        admin: &AccountHash,
+    ) -> (ContractHash, ContractPackageHash) {
+        let token_args = runtime_args! {
+            ARG_COLLECTION_NAME => TOKEN_CEP_78_NAME,
+            ARG_COLLECTION_SYMBOL => TOKEN_CEP_78_SYMBOL,
+            ARG_TOTAL_TOKEN_SUPPLY => 1000_u64,
+            ARG_OWNERSHIP_MODE => 2_u8, // must be transferable
```

Comment:
Right now script is just inputing an CEP-78 NFT with fixed modalities, no checks regarding transferability. We can see later how to add modalities to the auction contract it self and security checks.

Reply:
We can manually assess the modality combos and add tests that are expected to behave in an appropriate way depending on the combo. But it seems there is no way to interrogate the CEP-78 contract externally to check the modalities (on the other hand, some/all modalities will gracefully fail on auction install anyway)